src/screen/MyIdentityScreen/DisplayJudgement.tsx

```diff
@@ -41,7 +41,7 @@ function DisplayJudgement(props: PropTypes) {
   const {api} = useContext(ChainApiContext);
   const successMsg = `This address has ${judgementCount} judgement${
     judgementCount > 1 ? 's' : ''
-  } from Registrar ${detail.data?.judgements.map((judgement) => `#${judgement[0]}`).join(',')}. It's all set. 🎉`;
+  } from Registrar ${detail.data?.judgements?.map((judgement) => `#${judgement[0]}`).join(',')}. It's all set. 🎉`;
```

Comment:
how come judgements is nullable now? If that is the case the string message won't make sense no?

Reply:
it's not, don't know why I added that `?`.  it's now like before.